Brokerages/InteractiveBrokers/Client/OpenOrderEventArgs.cs

```diff
@@ -59,7 +59,7 @@ public OpenOrderEventArgs(int orderId, Contract contract, Order order, OrderStat
         /// </summary>
         public override string ToString()
         {
-            return $"OrderId: {OrderId}, Symbol: {Contract.Symbol}, OrderStatus: {OrderState.Status}";
+            return $"OrderId: {OrderId}, Contract: {Contract}, OrderStatus: {OrderState.Status}";
```

Comment:
What is the ToString() for `Contract` class?

Reply:
``` C#
        public override string ToString()
        {
            return SecType + " " + Symbol + " " + Currency + " " + Exchange;
        }
```